### ISSUE TITLE
[High]: Chrome+Jaws : Screen reader: Role and label are not defined for "Change default skin tone" control. 

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9667]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Navigate to the already created room from the Chat list using the Tab key and press Enter on it.
3. Navigate to the Emoji button located at the bottom of the page, inside the textfield.
4. Navigate to Emoji control using Tab key and press Enter to activate it.
5. A pop-up will appear.
6. Navigate to the "Change default skin tone" located at the bottom of the pop-up.
7. Observe the screen reader's behavior.

## Expected Result:
When a role is defined appropriately for the interactive element, the screen reader user will interact with the element effectively.

## Actual Result:
Role is not defined for the "Change default skin tone" interactive element. This made it difficult for screen reader users to access the functionality associated with it. Additionally, the label is not defined for the mentioned interactive element in the source code of the page.

HTML:
`<div dir="auto" class="css-901oao r-13qz1uu" style="background-color: rgb(255, 255, 255); color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 11px; font-weight: 700; padding-top: 4px; padding-bottom: 4px; text-align: left;">Change default skin tone</div>`

## Workaround:
Yes

## Area issue was found in:
Emoji pop-up

## Failed WCAG checkpoints
4.1.2

## User impact:
Screen reader users will not be made aware that these elements are buttons.

## Suggested resolution:
Ensure that all the page functionality is available for all users irrespective of the device being used.
Apply the following changes:

Specify role="button" for the 'div' element containing the "Change default skin tone" control.
Provide aria-label value such as "back" for the 'div' element containing the "Change default skin tone" control.

## Platform:
- Web
- Desktop App

**Version Number:*v1.1.79-9*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![change default](https://user-images.githubusercontent.com/98817464/176918441-ff47568c-6f50-483c-a653-47540cfbcc09.jpg)
